client/next-prefetcher.js

```diff
@@ -1,13 +1,14 @@
 /* global self */
 
 const CACHE_NAME = 'next-prefetcher-v1'
+const log = () => {}
```

Comment:
Still, logging is pretty useful when debugging. 
So, we keep a log function like this and we can change it to `console.log.bind(console)` when we need to do some logging.